vevestaX/vevesta.py

```diff
@@ -432,11 +435,10 @@ def __EDA(self, fileName):
                 nonNumericPlotSheet.add_image(nonNumericFeatureImage)
 
             if os.path.exists(os.path.join(directoryToDumpData,FeatureHistogramImageFile)):
-                workBook.create_sheet('EDA-Feature Histogram')
-                featureDistribution = workBook['EDA-Feature Histogram']
+                workBookName = workBook.create_sheet('EDA-Feature Histogram')
                 featureDistributionImage = openpyxl.drawing.image.Image(os.path.join(directoryToDumpData,FeatureHistogramImageFile))
                 featureDistributionImage.anchor = columnTextImgone
-                featureDistribution.add_image(featureDistributionImage)
```

Comment:
This is all wrong. the string ."EDA-Feature Histogram" had to be called workBookName